src/ert/ensemble_evaluator/dispatch.py

```diff
@@ -74,20 +60,47 @@ def done_logger(_):
         events_handling.add_done_callback(done_logger)
         await events_handling
 
+    def run_dispatcher(self, loop):
+        try:
+            loop.run_until_complete(self._job())
+        except asyncio.CancelledError as ex:
+            logger.warning(f"batcher was cancelled: {ex}")
+        except Exception as failure:  # pylint: disable=broad-exception-caught
+            logger.error(f"exception in batcher: {failure}")
```

Comment:
I would perhaps rephrase it `Exception in run disbatcher event loop` to indicate where this was cauth?

Reply:
personally, i think `event loop` is a technical detail that does not add much valuable information (we also get a stack trace, after all), so i would skip that part. we can change `batcher` to `batching dispatcher` though. and `logger.error` should be `logger.exception` I think :thinking: